libs/ui-components/src/lib/AddressLink.tsx

```diff
@@ -1,4 +1,5 @@
 import { EXPLORER_URLS, UNICODE_SYMBOLS } from 'libs/util-constants/src';
+// eslint-disable-next-line @nx/enforce-module-boundaries
```

Comment:
it's not ideal, but unfortunately I didn't have time to fix it properly, it's a global problem in mono repo, gonna bring it up on the next eng meeting..

Reply:
added to the doc, can't remember what the other item was, will DM